### ISSUE TITLE
error in demo-gatsby after npm run start

### ISSUE THREAD

**Issue by burasuk** [tinacms/tinacms#470]
### Steps to reproduce

```
git clone git@github.com:tinacms/tinacms.git
cd tinacms
npm install && npm run bootstrap
npm run build

cd packages/demo-gatsby
npm run start
```

open browser

### Expected result

Working demo

### Actual result

Error page with statement "TypeError: Cannot read property 'firstName' of null"

![image](https://user-images.githubusercontent.com/7629077/68835536-96aeab00-06b8-11ea-8812-deb12e01ce41.png)

### Environment

```
"name": "demo-gatsby",
 "version": "1.1.5",
```

any browser

**Comment by spbyrne** [comment]
@burasuk I'm unable to recreate this issue; things seem to be working for the demo. That error sounds like the author data file was missing, but it's in the repo. What version of node are you running?

**Comment by mkozachok** [comment]
I have the error.

**Comment by ncphillips** [comment]
I added the `needs-test-case` label because we can't reproduce the problem. Is there more information that you can give us? i.e. node version, operating system, etc.

**Comment by spbyrne** [comment]
I believe this is closed with [fix: add windows slash fix to gatsby json plugin #493](https://github.com/tinacms/tinacms/pull/493), since that exact error came up for me on windows before the fix.
